project-2-e-commerce/src/component/cart/CartItem.js

```diff
@@ -26,13 +22,14 @@ function CartItem({ item, getCartDelete, changeIsChange }) {
                 break;
         }
         if (isLogged) {
-            const newCart = {...item, quantity: value}
-            dispatch(updateUserCart(userData.id, newCart))
+            const newLocalCarts = updateCartList(id, value, userData.carts)
+            const newUserData = {...userData, carts: newLocalCarts}
+            localStorage.setItem('user-login', JSON.stringify(newUserData))
         } else {
             const newLocalCarts = updateCartList(id, value, localCarts)
             localStorage.setItem('local-carts', JSON.stringify(newLocalCarts))
-            changeIsChange()
         }
+        changeIsChange()
```

Comment:
```suggestion
        toggle...()
```
Đặt tên có ý nghĩa hơn chút nhé, để tên như này chắc biết làm hàm thực hiện chức năng gì đâu.